packages/components/psammead-most-read/src/utilities/rankMinWidth.js

```diff
@@ -0,0 +1,59 @@
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+  GEL_SPACING_QUAD,
+  GEL_SPACING_QUIN,
+  GEL_SPACING_SEXT,
+} from '@bbc/gel-foundations/spacings';
+
+// Services with fonts that have glyphs thinner than the majority of other fonts.
+// This was mainly based on the old overrides (ie. Any group0 value < 2rem).
+export const thinFontServices = [
+  'arabic',
+  'bengali',
+  'pashto',
+  'persian',
+  'tamil',
+  'telegu',
+  'urdu',
+];
+
+// If numberOfItems < 10, no extra spacing needs to be accounted for.
+export const singleDigitDefault = {
+  group0: GEL_SPACING_TRPL,
+  group1: GEL_SPACING_TRPL,
+  group2: GEL_SPACING_TRPL,
+  group3: GEL_SPACING_QUAD,
+  group5: GEL_SPACING_QUAD,
+};
+
+// If numberOfItems >= 10, extra spacing needs to be accounted for.
+export const doubleDigitDefault = {
+  group3: GEL_SPACING_QUAD,
+  group5: GEL_SPACING_QUAD,
+  // These values are used to align the rank when a double digit exists in the column
+  group0_1_column: GEL_SPACING_QUIN,
```

Comment:
Could these be `group1WithOneColumn`, `group3WithTwoColumns`, `group5WithFiveColumns` ? The current names don't give away the intent very well, and maybe plain english makes more sense (and keeping to camelcase convention).

Reply:
Yep that's much better